### ISSUE TITLE
Model list on `https://huggingface.co/second-state`

### ISSUE THREAD

**Issue by apepkuss** [LlamaEdge/LlamaEdge#71]
### Summary

- Provide k-quant models
- Maintain existing gguf models
  - Embedding models
    - [x] [second-state/Nomic-embed-text-v1.5-Embedding-GGUF](https://huggingface.co/second-state/Nomic-embed-text-v1.5-Embedding-GGUF)
    - [x] [second-state/All-MiniLM-L6-v2-Embedding-GGUF](https://huggingface.co/second-state/All-MiniLM-L6-v2-Embedding-GGUF)
    - [x] [second-state/E5-Mistral-7B-Instruct-Embedding-GGUF](https://huggingface.co/second-state/E5-Mistral-7B-Instruct-Embedding-GGUF)
  
  - Multi-modal models
    - [ ] [second-state/MiniCPM-V-2_6-GGUF](https://huggingface.co/second-state/MiniCPM-V-2_6-GGUF)
    - [ ] [second-state/MiniCPM-Llama3-V-2_5-GGUF](https://huggingface.co/second-state/MiniCPM-Llama3-V-2_5-GGUF)
    - [ ] Qwen/Qwen-VL-Chat (Blocked by llama.cpp [#5331](https://github.com/ggerganov/llama.cpp/issues/5331))
    - [x] [second-state/Liberated-Qwen1.5-72B-GGUF](https://huggingface.co/second-state/Liberated-Qwen1.5-72B-GGUF)
    - [ ] second-state/Yi-VL-6B
    - [ ] second-state/Yi-VL-34B
    - [x] [second-state/Llava-v1.6-Vicuna-7B-GGUF](https://huggingface.co/second-state/Llava-v1.6-Vicuna-7B-GGUF)
    - [x] [second-state/Llava-v1.5-7B-GGUF](https://huggingface.co/second-state/Llava-v1.5-7B-GGUF)
  
  - Text2Image
    - [x] [second-state/FLUX.1-dev-GGUF](https://huggingface.co/second-state/FLUX.1-dev-GGUF)
    - [ ] [second-state/3dAnimationDiffusion_v10-GGUF](https://huggingface.co/second-state/3dAnimationDiffusion_v10-GGUF)
    - [x] [second-state/FLUX.1-schnell-GGUF](https://huggingface.co/second-state/FLUX.1-schnell-GGUF)
    - [x] [second-state/stable-diffusion-3-medium-GGUF](https://huggingface.co/second-state/stable-diffusion-3-medium-GGUF)
    - [x] [second-state/stable-diffusion-2-1-GGUF](https://huggingface.co/second-state/stable-diffusion-2-1-GGUF)
    - [x] [second-state/stable-diffusion-v1-5-GGUF](https://huggingface.co/second-state/stable-diffusion-v1-5-GGUF)
    - [x] [second-state/stable-diffusion-v-1-4-GGUF](https://huggingface.co/second-state/stable-diffusion-v-1-4-GGUF)

  - Chat/instruct models
    - [x] [second-state/internlm2_5-7b-chat-1m-GGUF](https://huggingface.co/second-state/internlm2_5-7b-chat-1m-GGUF)
    - [ ] [second-state/DeepSeek-V2.5-GGUF](https://huggingface.co/second-state/DeepSeek-V2.5-GGUF)
    - [x] [second-state/datagemma-rag-27b-it-GGUF](https://huggingface.co/second-state/datagemma-rag-27b-it-GGUF)
    - [x] [second-state/datagemma-rig-27b-it-GGUF](https://huggingface.co/second-state/datagemma-rig-27b-it-GGUF)
    - [ ] [second-state/c4ai-command-r-08-2024-GGUF](https://huggingface.co/second-state/c4ai-command-r-08-2024-GGUF)
    - [x] [second-state/Llama-3-Taiwan-8B-Instruct-GGUF](https://huggingface.co/second-state/Llama-3-Taiwan-8B-Instruct-GGUF)
    - [x] [second-state/Llama-3-Taiwan-70B-Instruct-GGUF](https://huggingface.co/second-state/Llama-3-Taiwan-70B-Instruct-GGUF)
    - [x] [second-state/Infinity-Instruct-7M-Gen-Llama3_1-70B-GGUF](https://huggingface.co/second-state/Infinity-Instruct-7M-Gen-Llama3_1-70B-GGUF)
    - [x] [second-state/Infinity-Instruct-7M-Gen-Llama3_1-8B-GGUF](https://huggingface.co/second-state/Infinity-Instruct-7M-Gen-Llama3_1-8B-GGUF)
    - [x] [second-state/Infinity-Instruct-7M-Gen-mistral-7B-GGUF](https://huggingface.co/second-state/Infinity-Instruct-7M-Gen-mistral-7B-GGUF)
    - [x] [second-state/Qwen2-Math-1.5B-Instruct-GGUF](https://huggingface.co/second-state/Qwen2-Math-1.5B-Instruct-GGUF)
    - [x] [second-state/Qwen2-Math-7B-Instruct-GGUF](https://huggingface.co/second-state/Qwen2-Math-7B-Instruct-GGUF)
    - [x] [second-state/Qwen2-Math-72B-Instruct-GGUF](https://huggingface.co/second-state/Qwen2-Math-72B-Instruct-GGUF)
    - [x] [second-state/Palmyra-Med-70B-32K-GGUF](https://huggingface.co/second-state/Palmyra-Med-70B-32K-GGUF)
    - [x] [second-state/Palmyra-Fin-70B-32K-GGUF](https://huggingface.co/second-state/Palmyra-Fin-70B-32K-GGUF)
    - [x] [second-state/internlm2_5-1_8b-chat-GGUF](https://huggingface.co/second-state/internlm2_5-1_8b-chat-GGUF)
    - [x] [second-state/internlm2_5-20b-chat-GGUF](https://huggingface.co/second-state/internlm2_5-20b-chat-GGUF)
    - [x] [second-state/DeepSeek-Coder-V2-Instruct-GGUF](https://huggingface.co/second-state/DeepSeek-Coder-V2-Instruct-GGUF)
    - [x] [second-state/Mistral-Large-Instruct-2407-GGUF](https://huggingface.co/second-state/Mistral-Large-Instruct-2407-GGUF)
    - [x] [second-state/gemma-2-2b-it-GGUF](https://huggingface.co/second-state/gemma-2-2b-it-GGUF)
    - [x] [second-state/Llama-3-Groq-70B-Tool-Use-GGUF](https://huggingface.co/second-state/Llama-3-Groq-70B-Tool-Use-GGUF)
    - [x] [second-state/Llama3.1-8B-Chinese-Chat-GGUF](https://huggingface.co/second-state/Llama3.1-8B-Chinese-Chat-GGUF)
    - [x] [second-state/SmolLM-360M-Instruct-GGUF](https://huggingface.co/second-state/SmolLM-360M-Instruct-GGUF)
    - [x] [second-state/SmolLM-135M-Instruct-GGUF](https://huggingface.co/second-state/SmolLM-135M-Instruct-GGUF)
    - [x] [second-state/Meta-Llama-3.1-70B-Instruct-GGUF](https://huggingface.co/second-state/Meta-Llama-3.1-70B-Instruct-GGUF)
    - [x] [second-state/Meta-Llama-3.1-8B-Instruct-GGUF](https://huggingface.co/second-state/Meta-Llama-3.1-8B-Instruct-GGUF)
    - [x] [second-state/Llama-3-Groq-8B-Tool-Use-GGUF](https://huggingface.co/second-state/Llama-3-Groq-8B-Tool-Use-GGUF)
    - [x] [second-state/Hermes-2-Pro-Llama-3-8B-GGUF](https://huggingface.co/second-state/Hermes-2-Pro-Llama-3-8B-GGUF)
    - [x] [second-state/Mistral-Nemo-Instruct-2407-GGUF](https://huggingface.co/second-state/Mistral-Nemo-Instruct-2407-GGUF)
    - [x] [second-state/mathstral-7B-v0.1-GGUF](https://huggingface.co/second-state/mathstral-7B-v0.1-GGUF)
    - [x] [second-state/stablelm-2-12b-chat-GGUF](https://huggingface.co/second-state/stablelm-2-12b-chat-GGUF)
    - [x] [second-state/DeepSeek-V2-Lite-Chat-GGUF](https://huggingface.co/second-state/DeepSeek-V2-Lite-Chat-GGUF)
    - [x] [second-state/glm-4-9b-chat-1m-GGUF](https://huggingface.co/second-state/glm-4-9b-chat-1m-GGUF)
    - [x] [second-state/glm-4-9b-chat-GGUF](https://huggingface.co/second-state/glm-4-9b-chat-GGUF)
    - [x] [second-state/gte-Qwen2-1.5B-instruct-GGUF](https://huggingface.co/second-state/gte-Qwen2-1.5B-instruct-GGUF)
    - [x] [second-state/Gemma-2-9B-Chinese-Chat-GGUF](https://huggingface.co/second-state/Gemma-2-9B-Chinese-Chat-GGUF)
    - [x] [second-state/gemma-2-27b-it-GGUF](https://huggingface.co/second-state/gemma-2-27b-it-GGUF)
    - [x] [second-state/gemma-2-9b-it-GGUF](https://huggingface.co/second-state/gemma-2-9b-it-GGUF)
    - [ ] [second-state/DeepSeek-Coder-V2-Lite-Instruct-GGUF](https://huggingface.co/second-state/DeepSeek-Coder-V2-Lite-Instruct-GGUF)
    - [x] [second-state/Qwen2-72B-Instruct-GGUF](https://huggingface.co/second-state/Qwen2-72B-Instruct-GGUF)
    - [x] [second-state/Qwen2-7B-Instruct-GGUF](https://huggingface.co/second-state/Qwen2-7B-Instruct-GGUF)
    - [x] [second-state/Qwen2-1.5B-Instruct-GGUF](https://huggingface.co/second-state/Qwen2-1.5B-Instruct-GGUF)
    - [x] [second-state/Qwen2-0.5B-Instruct-GGUF](https://huggingface.co/second-state/Qwen2-0.5B-Instruct-GGUF)
    - [x] [second-state/Mistral-7B-Instruct-v0.3-GGUF](https://huggingface.co/second-state/Mistral-7B-Instruct-v0.3-GGUF)
    - [x] [second-state/Codestral-22B-v0.1-GGUF](https://huggingface.co/second-state/Codestral-22B-v0.1-GGUF)
    - [x] [second-state/Llama-3-Instruct-8B-SimPO-GGUF](https://huggingface.co/second-state/Llama-3-Instruct-8B-SimPO-GGUF)
    - [x] [second-state/Phi-3-medium-128k-instruct-GGUF](https://huggingface.co/second-state/Phi-3-medium-128k-instruct-GGUF)
    - [x] [second-state/Phi-3-mini-128k-instruct-GGUF](https://huggingface.co/second-state/Phi-3-mini-128k-instruct-GGUF)
    - [x] [second-state/Phi-3-medium-4k-instruct-GGUF](https://huggingface.co/second-state/Phi-3-medium-4k-instruct-GGUF)
    - [x] [second-state/Yi-1.5-9B-Chat-16K-GGUF](https://huggingface.co/second-state/Yi-1.5-9B-Chat-16K-GGUF)
    - [x] [second-state/Yi-1.5-34B-Chat-16K-GGUF](https://huggingface.co/second-state/Yi-1.5-34B-Chat-16K-GGUF)
    - [x] [second-state/Yi-1.5-34B-Chat-GGUF](https://huggingface.co/second-state/Yi-1.5-34B-Chat-GGUF)
    - [x] [second-state/Yi-1.5-9B-Chat-GGUF](https://huggingface.co/second-state/Yi-1.5-9B-Chat-GGUF)
    - [x] [second-state/Yi-1.5-6B-Chat-GGUF](https://huggingface.co/second-state/Yi-1.5-6B-Chat-GGUF)
    - [x] [second-state/Llama-3-8B-Japanese-Instruct-GGUF](https://huggingface.co/second-state/Llama-3-8B-Japanese-Instruct-GGUF)
    - [ ] [second-state/StarCoder2-15B-Instruct-v0.1-GGUF](https://huggingface.co/second-state/StarCoder2-15B-Instruct-v0.1-GGUF)
    - [x] [second-state/Llama-3-8B-Instruct-Gradient-1048k-GGUF](https://huggingface.co/second-state/Llama-3-8B-Instruct-Gradient-1048k-GGUF)
    - [x] [second-state/Llama3-8B-Chinese-Chat-GGUF](https://huggingface.co/second-state/Llama3-8B-Chinese-Chat-GGUF)
    - [x] [second-state/Llama-3-8B-Instruct-262k-GGUF](https://huggingface.co/second-state/Llama-3-8B-Instruct-262k-GGUF) 
    - [x] [second-state/Phi-3-mini-4k-instruct-GGUF](https://huggingface.co/second-state/Phi-3-mini-4k-instruct-GGUF)
    - [x] [second-state/Llama-3-8B-Instruct-GGUF](https://huggingface.co/second-state/Llama-3-8B-Instruct-GGUF)
    - [x] [second-state/Meta-Llama-3-70B-Instruct-GGUF](second-state/Meta-Llama-3-70B-Instruct-GGUF)
    - [x] [second-state/CodeQwen1.5-7B-Chat-GGUF](https://huggingface.co/second-state/CodeQwen1.5-7B-Chat-GGUF)
    - [x] [second-state/CodeGemma-7b-it-GGUF](https://huggingface.co/second-state/CodeGemma-7b-it-GGUF)
    - [x] [second-state/gemma-1.1-2b-it-GGUF](https://huggingface.co/second-state/gemma-1.1-2b-it-GGUF)
    - [x] [second-state/gemma-1.1-7b-it-GGUF](https://huggingface.co/second-state/gemma-1.1-7b-it-GGUF)
    - [x] [second-state/Octopus-v2-GGUF](https://huggingface.co/second-state/Octopus-v2-GGUF)
    - [ ] [second-state/C4AI-Command-R-v01-GGUF](https://huggingface.co/second-state/C4AI-Command-R-v01-GGUF)
    - [ ] [second-state/ChatAllInOne-Yi-34B-200K-V1-GGUF](https://huggingface.co/second-state/ChatAllInOne-Yi-34B-200K-V1-GGUF)
    - [ ] [second-state/StarCoder2-3B-GGUF](https://huggingface.co/second-state/StarCoder2-3B-GGUF)
    - [ ] [second-state/StarCoder2-7B-GGUF](https://huggingface.co/second-state/StarCoder2-7B-GGUF)
    - [ ] [second-state/StarCoder2-15B-GGUF](https://huggingface.co/second-state/StarCoder2-15B-GGUF)
    - [x] [second-state/Gemma-7b-it-GGUF](https://huggingface.co/second-state/Gemma-7b-it-GGUF)
    - [x] [second-state/Gemma-2b-it-GGUF](https://huggingface.co/second-state/Gemma-2b-it-GGUF)
    - [x] [second-state/Yi-6B-Chat-GGUF](https://huggingface.co/second-state/Yi-6B-Chat-GGUF)
    - [x] [second-state/Qwen1.5-0.5B-Chat-GGUF](https://huggingface.co/second-state/Qwen1.5-0.5B-Chat-GGUF)
    - [x] [second-state/Qwen1.5-1.8B-Chat-GGUF](https://huggingface.co/second-state/Qwen1.5-1.8B-Chat-GGUF)
    - [x] [second-state/Qwen1.5-4B-Chat-GGUF](https://huggingface.co/second-state/Qwen1.5-4B-Chat-GGUF)
    - [x] [second-state/Qwen1.5-7B-Chat-GGUF](https://huggingface.co/second-state/Qwen1.5-7B-Chat-GGUF)
    - [x] [second-state/Qwen1.5-14B-Chat-GGUF](https://huggingface.co/second-state/Qwen1.5-14B-Chat-GGUF)
    - [x] [second-state/CodeLlama-70b-Instruct-hf-GGUF](https://huggingface.co/second-state/CodeLlama-70b-Instruct-hf-GGUF)
    - [x] [second-state/Llama-2-7B-Chat-GGUF](https://huggingface.co/second-state/Llama-2-7B-Chat-GGUF) 
    - [x] [second-state/Llama-2-13B-Chat-GGUF](https://huggingface.co/second-state/Llama-2-13B-Chat-GGUF)
    - [x] [second-state/OpenChat-3.5-0106-GGUF](https://huggingface.co/second-state/OpenChat-3.5-0106-GGUF)
    - [x] [second-state/openchat-3.5-1210-GGUF](https://huggingface.co/second-state/openchat-3.5-1210-GGUF)
    - [x] [second-state/OpenChat-3.5-GGUF](https://huggingface.co/second-state/OpenChat-3.5-GGUF)
    - [x] [second-state/OpenHermes-2.5-Mistral-7B-GGUF](https://huggingface.co/second-state/OpenHermes-2.5-Mistral-7B-GGUF)
    - [x] [second-state/Orca-2-13B-GGUF](https://huggingface.co/second-state/Orca-2-13B-GGUF)
    - [x] [second-state/TinyLlama-1.1B-Chat-v1.0-GGUF](https://huggingface.co/second-state/TinyLlama-1.1B-Chat-v1.0-GGUF)
    - [x] [second-state/Baichuan2-13B-Chat-GGUF](https://huggingface.co/second-state/Baichuan2-13B-Chat-GGUF)
    - [x] [second-state/Baichuan2-7B-Chat-GGUF](https://huggingface.co/second-state/Baichuan2-7B-Chat-GGUF)
    - [x] [second-state/dolphin-2.6-mistral-7B-GGUF](https://huggingface.co/second-state/dolphin-2.6-mistral-7B-GGUF)
    - [x] [second-state/Dolphin-2.7-mixtral-8x7b-GGUF](https://huggingface.co/second-state/Dolphin-2.7-mixtral-8x7b-GGUF)
    - [x] [second-state/phi-2-GGUF](https://huggingface.co/second-state/phi-2-GGUF)
    - [x] [second-state/SOLAR-10.7B-Instruct-v1.0-GGUF](https://huggingface.co/second-state/SOLAR-10.7B-Instruct-v1.0-GGUF)
    - [x] [second-state/Starling-LM-7B-alpha-GGUF](https://huggingface.co/second-state/Starling-LM-7B-alpha-GGUF)
    - [x] [second-state/Deepseek-Coder-6.7B-Instruct-GGUF](https://huggingface.co/second-state/Deepseek-Coder-6.7B-Instruct-GGUF) 
    - [x] [second-state/Deepseek-LLM-7B-Chat-GGUF](https://huggingface.co/second-state/Deepseek-LLM-7B-Chat-GGUF)
    - [x] [second-state/Mistral-7B-Instruct-v0.2-GGUF](https://huggingface.co/second-state/Mistral-7B-Instruct-v0.2-GGUF)
    - [x] [second-state/Mistral-7B-Instruct-v0.1-GGUF](https://huggingface.co/second-state/Mistral-7B-Instruct-v0.1-GGUF)
    - [x] [second-state/MistralLite-7B-GGUF](https://huggingface.co/second-state/MistralLite-7B-GGUF)
    - [x] [second-state/Zephyr-7B-Beta-GGUF](https://huggingface.co/second-state/Zephyr-7B-Beta-GGUF)
    - [x] [second-state/Zephyr-7B-Alpha-GGUF](https://huggingface.co/second-state/Zephyr-7B-Alpha-GGUF)
    - [x] [second-state/Neural-Chat-7B-v3-3-GGUF](https://huggingface.co/second-state/Neural-Chat-7B-v3-3-GGUF)
    - [x] [second-state/Neural-Chat-7B-v3-1-GGUF](https://huggingface.co/second-state/Neural-Chat-7B-v3-1-GGUF)
    - [x] [second-state/CodeLlama-13B-Instruct-GGUF](https://huggingface.co/second-state/CodeLlama-13B-Instruct-GGUF)
    - [x] [second-state/ELYZA-japanese-Llama-2-13b-fast-instruct-GGUF](https://huggingface.co/second-state/ELYZA-japanese-Llama-2-13b-fast-instruct-GGUF)
    - [x] [second-state/WizardLM-13B-V1.0-Uncensored-GGUF](https://huggingface.co/second-state/WizardLM-13B-V1.0-Uncensored-GGUF)
    - [x] [second-state/Wizard-Vicuna-13B-Uncensored-GGUF](https://huggingface.co/second-state/Wizard-Vicuna-13B-Uncensored-GGUF)
    - [x] [second-state/WizardCoder-Python-7B-v1.0-GGUF](https://huggingface.co/second-state/WizardCoder-Python-7B-v1.0-GGUF)
    - [x] [second-state/Nous-Hermes-2-Mixtral-8x7B-SFT-GGUF](https://huggingface.co/second-state/Nous-Hermes-2-Mixtral-8x7B-SFT-GGUF)
    - [x] [second-state/Nous-Hermes-2-Mixtral-8x7B-DPO-GGUF](https://huggingface.co/second-state/Nous-Hermes-2-Mixtral-8x7B-DPO-GGUF)
    - [x] [second-state/stablelm-2-zephyr-1.6b-GGUF](https://huggingface.co/second-state/stablelm-2-zephyr-1.6b-GGUF)
    - [x] [second-state/Yi-34B-Chat-GGUF](https://huggingface.co/second-state/Yi-34B-Chat-GGUF)
    - [x] [second-state/Yi-34Bx2-MoE-60B-GGUF](https://huggingface.co/second-state/Yi-34Bx2-MoE-60B-GGUF)
    - [x] [second-state/Dolphin-2.2-Yi-34B-GGUF](https://huggingface.co/second-state/Dolphin-2.2-Yi-34B-GGUF)
    - [x] [second-state/Samantha-1.2-Mistral-7B-GGUF](https://huggingface.co/second-state/Samantha-1.2-Mistral-7B-GGUF)
    - [x] [second-state/Samantha-1.11-CodeLlama-34B-GGUF](https://huggingface.co/second-state/Samantha-1.11-CodeLlama-34B-GGUF)
    - [x] [second-state/Mixtral-8x7B-Instruct-v0.1-GGUF](https://huggingface.co/second-state/Mixtral-8x7B-Instruct-v0.1-GGUF)
    - [x] [second-state/NeuralBeagle14-7B-GGUF](https://huggingface.co/second-state/NeuralBeagle14-7B-GGUF)
    - [x] [second-state/OrionStar-Yi-34B-Chat-Llama-GGUF](https://huggingface.co/second-state/OrionStar-Yi-34B-Chat-Llama-GGUF)
  
  

- Models blocked by issues
  - [second-state/Calm2-7B-Chat-GGUF](https://huggingface.co/second-state/Calm2-7B-Chat-GGUF) (Blocked by [llama.cpp #774](https://github.com/ggerganov/llama.cpp/issues/774))
  - [second-state/CausalLM-14B-GGUF](https://huggingface.co/second-state/CausalLM-14B-GGUF) (Blocked by [llama.cpp #4958](https://github.com/ggerganov/llama.cpp/issues/4958) and [llama.cpp #3732](https://github.com/ggerganov/llama.cpp/issues/3732))
  - [second-state/WizardCoder-33B-V1.1-GGUF](https://huggingface.co/second-state/WizardCoder-33B-V1.1-GGUF) (Blocked by [llama.cpp #774](https://github.com/ggerganov/llama.cpp/issues/774))

**Comment by cyberluke** [comment]
what about [second-state/DeepSeek-V2.5-GGUF](https://huggingface.co/second-state/DeepSeek-V2.5-GGUF) ?

**Comment by apepkuss** [comment]
@cyberluke The model card of second-state/DeepSeek-V2.5-GGUF has been updated. You can try it with the latest version of LlamaEdge: https://github.com/LlamaEdge/LlamaEdge/releases/tag/0.14.11. The model is so large, even in the gguf version, that we don't have enough resources to verify it now. Please open a new issue if something wrong. Thanks a lot!
